src/classes/CRLP_Rollup_SEL.cls

```diff
@@ -46,21 +46,39 @@ public class CRLP_Rollup_SEL {
         get {
             if (cachedRollups == null) {
                 cachedRollups = [
-                        SELECT Id, DeveloperName, MasterLabel, Active__c, Use_Fiscal_Year__c, Filter_Group__r.DeveloperName,
-                                Description__c, Filter_Group__c, Filter_Group__r.MasterLabel, Operation__c, Integer__c,
-                                Time_Bound_Operation_Type__c
-                                , Summary_Field__r.QualifiedApiName, Summary_Field__r.Label
-                                , Summary_Object__r.QualifiedApiName, Summary_Object__r.Label
-                                , Detail_Field__r.QualifiedApiName, Detail_Field__r.Label
-                                , Detail_Object__r.QualifiedApiName, Detail_Object__r.Label
-                                , Amount_Field__r.QualifiedApiName, Amount_Field__r.Label
-                                , Amount_Object__r.QualifiedApiName, Amount_Object__r.Label
-                                , Date_Field__r.QualifiedApiName, Date_Field__r.Label
-                                , Date_Object__r.QualifiedApiName, Date_Object__r.Label
-                                , Is_Deleted__c
-                        FROM Rollup__mdt
-                        WHERE Is_Deleted__c = false
-                        ORDER BY Summary_Object__c, MasterLabel
+                    SELECT 
```

Comment:
-  Formatted SOQL for readability
-  Changed order of SELECT to group similar Fields together
    -  My preference is to put all of queried Object's Fields first then relationships' fields second (and sub-queries last if present)